src/foundry/common/types.mjs.d.ts

```diff
@@ -1,3 +1,86 @@
+interface DocumentModificationOptions {
```

Comment:
type `DocumentConstructionContext` is missing

Reply:
Forgot it in the shuffle